use-timescale/vpc.md

```diff
@@ -10,226 +10,218 @@ cloud_ui:
+       * `Info`: the CIDR block of your Timescale VPC.
+1.  Click `Add rule`, then click `Create security group`.
 
 </Procedure>
 
-## Migrating a VPC service between networks
+### Attach a Timescale Service to the Peering VPC
 
-In most cases, when you have connected a service to a VPC, you need to keep it
-attached to ensure that your applications continue to run without interruption.
-However, you can migrate Timescale services between VPCs within a project,
-or migrate them to and from the public network, if you need to.
+Now that Timescale is communicating securely with your AWS infrastructure, you can attach 
+one or more Timescale Services to the VPC. 
 
-<Highlight type="warning">
-Timescale uses a different DNS name for a Timescale service once it has
-been attached to a VPC. This means that you need to update your connection
-string if you are migrating a service between the public internet and a VPC.
-</Highlight>
+After you attach a Timescale Service to a VPC, you can only access it through the peered
+AWS VPC. It is no longer accessible using the public internet.
 
-Before you begin, ensure you already have your VPC connection set up.
 
 <Procedure>
 
-### Migrating Timescale services to or between VPCs
+1.  In [Timescale Console > Services][console-services] select the Service you want to
+    connect to the VPC. 
+   If you don't have a Timescale Service, [create a new one][create-service].
+1. Click `Operations` > `VPC`.
+1. Select the VPC, then click `Attach VPC`.
+
+</Procedure>
+
+And that is it, your Timescale Service is now securely communicating securely with your AWS
+account inside a VPC.
+
+## Migrate a Timescale Service between VPCs
 
-1.  [Log in to your Timescale account][cloud-login] and navigate to
-    the `Services` section. Click the name of the service you want to migrate.
-1.  In the `Operations` tab, navigate to the `VPC` section, and select the new
-    VPC to attach the service to. The migration can take a few minutes to
-    complete, and your services are not accessible during this time.
+To ensure that your applications continue to run without interruption, you keep 
+Timescale Services attached to the VPC. However, you can change the VPN your 
+Timescale Service is attached to, or disconnect from a VPC and enable access to the 
+Timescale Service from the public internet.
 
-<Highlight type="important">
-Migrating your services to a VPC requires a change to the DNS settings for the
-service. If you receive a DNS error, allow some more time for DNS propagation to
-complete.
+<Highlight type="info">
+Timescale uses a different DNS for Timescale Services that are attached to a VPC.
+When you migrate a Timescale Service between public access and a VPC, you need
+to update your connection string.
 </Highlight>
 
+<Procedure>
+
+1.  In [Timescale Console > Services][console-services] select the Service to migrate.
+    If you don't have a Timescale Service, [create a new one][create-service].
```

Comment:
I don't think it makes sense to ask people to create a service if they don't have one in this section since these are the instructions to migrate an existing service